tests/phpunit/includes/handle-wp-redirect-trait.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Trait to run tests containing `wp_redirect()` and `exit`.
+ * Also privides some assertions for the redirection.
+ */
+trait PLL_Handle_WP_Redirect_Trait {
+	/**
+	 * Whether or not `wp_redirect()` has been called.
+	 *
+	 * @var bool
+	 */
+	protected $has_redirect = false;
+
+	/**
+	 * Stores the last redirection location made with `wp_redirect()`.
+	 *
+	 * @var string
+	 */
+	protected $last_location;
```

Comment:
```suggestion
	public $redirect_location = '';
```
Making this property public would allow to avoid writing `assert_redirect_location()` which doesn't bring much in my opinion. That said if you prefer to have this method, you can keep it.